pkgs/applications/audio/plexamp/linux.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchurl, appimageTools, pkgs, pname, version, meta, ... }:
+
+let
+  inherit pname version meta;
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-${version}.AppImage";
+    name = "${pname}-${version}.AppImage";
+    sha512 = "7NQmH42yzItReQ5WPdcbNPr/xed74H4UyDNlX5PGmoJRBpV1RdeuW1KRweIWfYNhw0KeqULVTjr/aCggoWp4WA==";
```

Comment:
please use sha256 sri encoded hashes

Reply:
update-plexamp.sh is used to update the sha512 in default.nix. So  if i change to sha 256 , update-plexamp.sh could remove